test/test_exporter.py

```diff
@@ -158,11 +174,11 @@ def test_with_prefix(self, mock_post):
     def test_with_attributes(self, mock_post):
```

Comment:
```suggestion
    def test_with_default_dimensions(self, mock_post):
```

Reply:
Do we also have a test with data attributes (where the attributes are on the data point)?